<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demoQClass</class>
 <widget class="QMainWindow" name="demoQClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>834</width>
    <height>612</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>834</width>
    <height>673</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Quwi 0.1.295</string>
  </property>
  <property name="windowIcon">
   <iconset resource="demoq.qrc">
    <normaloff>:/demoQ/Resources/images/mainicon.ico</normaloff>:/demoQ/Resources/images/mainicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 93, 163);

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgb(23, 69, 136);
border-color: rgb(49, 98, 170);
border-width: 2px; 
border-style: solid;
border-radius: 6px;
color:white;

}
QPushButton{
color: white;
background-color: rgb(88, 139, 214);
border-color:  rgb(88, 139, 214);
border-width: 2px; 
border-style: solid;
border-radius: 6px;
}

QPushButton:hover{
background-color: rgb(116, 148, 195);
border-color: rgb(116, 148, 195);
}

QLabel#title_label{
background-color: rgb(255, 255, 255);
}
QLabel#icon_label{
background-color: rgb(255, 255, 255);
}
QFrame#title_frame{
background-color: rgb(255, 255, 255);
border-width: 1px 1px 0px 1px; 
border-color: gray;
border-style: solid;
}
QPushButton#title_x
{
background-color: rgb(255, 255, 255);
border-width: 0px; 
border-radius: 0px;
color: gray;
}
QPushButton#title_x:hover
{
background-color: rgb(232, 17, 35);
border-width: 0px; 
border-radius: 0px;
color: white;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCustom_titlebar" name="title_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>4</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="0">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="0">
         <property name="horizontalSpacing">
          <number>7</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="icon_label">
           <property name="minimumSize">
            <size>
             <width>5</width>
             <height>5</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="demoq.qrc">:/demoQ/Resources/images/mainicon.ico</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="title_label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>3</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>500</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Quwi 0.1.295</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="title_x">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>41</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>41</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>âœ•</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>32</number>
      </property>
      <property name="leftMargin">
       <number>220</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>220</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="demoq.qrc">:/demoQ/Resources/images/QUWI.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QMarked_LineEdit" name="text_email">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>58</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QMarked_LineEdit" name="text_password">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>58</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLoading_PushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>58</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://quwi.com/signup&quot;&gt;&lt;span style=&quot; color:#688fcf;text-decoration:none;&quot;&gt;Don't have an account? Register for free&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>24</number>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMarked_LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qmarked_lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QLoading_PushButton</class>
   <extends>QPushButton</extends>
   <header>qloading_pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCustom_titlebar</class>
   <extends>QFrame</extends>
   <header>qcustom_titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="demoq.qrc"/>
 </resources>
 <connections/>
</ui>
